html_xblock/html_xblock.py

```diff
@@ -222,3 +229,28 @@ def get_editable_fields(self):
                 fields.append(field_info)
 
         return fields
+
+    @staticmethod
+    def _get_statici18n_js_url():
```

Comment:
This should be added to xblock-utils if possible -- it's [copied from xblock-drag-and-drop](https://github.com/edx-solutions/xblock-drag-and-drop-v2/blob/master/drag_and_drop_v2/drag_and_drop_v2.py#L259-L273).

Also need to:
* add `@XBlock.needs('i18n')` to the `HTML5XBlock`
* use `loader.render_django_template`, and pass `self.runtime.i18n_service` into that function, so that this XBlock's own translations are used when translating strings in the templates.

cf https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/154